docs/diagrams/ModelClassDiagram.puml

```diff
@@ -36,6 +37,7 @@ ShowList *--> "1" UniqueShowList
 UniqueShowList --> "~* all" Show
 Show *--> "1" Name
 Show *--> "1" Status
+Show *--> "1 ... 0" Comment
```

Comment:
Shouldn't this be `Show *--> "0 ... 1"` ?

Reply:
Edited! Thanks for spotting the error!

Would update the picture according!